rest/src/main.rs

```diff
@@ -0,0 +1,56 @@
+use actix_web::{get, web, App, HttpResponse, HttpServer, Responder};
+use mbus_api::*;
+use structopt::StructOpt;
+
+const REST_VERSION: &str = "v0";
```

Comment:
This version can be determined from the Toml file.

Reply:
yeah good point. Not sure how we want to version the rest api tough, there seem to be 2 main ways of doing it, on the URI itself or through an http header.

Reply:
Yeah, good question, i *think* i prefer the version in the URI, but ... @jkryl  @mtzaurus what do you think?

Reply:
:+1: for uri, it is simpler for developers to keep track of what they are calling. Especially when consumed by outside of the team.

Reply:
Having API version encoded in URL is more convenient, IMO.